pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -40,7 +41,10 @@ patchShebangs() {
     local oldInterpreterLine
     local newInterpreterLine
 
-    [ -e "$dir" ] || return 0
+    if [ $# -eq 0 ]; then
+        echo "No arguments supplied to patchShebangs" >&2
+        exit 0
```

Comment:
This should be >0 so bash recognizes it as an error

Reply:
Should be return actually

Reply:
Why `>0`? That just pipes the string to a file with name "0"?

Reply:
Oops meant "greater than 0". That is non-zero code.

Reply:
ohhhh